libraries/networking/src/udt/SendQueue.h

```diff
@@ -79,17 +96,17 @@ private slots:
     SequenceNumber getNextSequenceNumber();
     MessageNumber getNextMessageNumber();
     
-    mutable QReadWriteLock _packetsLock; // Protects the packets to be sent list.
+    mutable std::mutex _packetsLock; // Protects the packets to be sent list.
     std::list<std::unique_ptr<Packet>> _packets; // List of packets to be sent
     
     Socket* _socket { nullptr }; // Socket to send packet on
     HifiSockAddr _destination; // Destination addr
     
-    std::atomic<uint32_t> _lastACKSequenceNumber; // Last ACKed sequence number
+    std::atomic<uint32_t> _lastACKSequenceNumber { 0 }; // Last ACKed sequence number
```

Comment:
Yep, I am dumb. I was reading `std::atomic` docs and read that zero initialization is performed by default. For `int` values (or `uint32_t`) that doesn't mean that it is set to zero.